MM-4644 Upgrade Rate Limiting help text (#4276)
mattermost/docs: source/administration/config-settings.rst

Enable Rate Limiting
^^^^^^^^^^^^^^^^^^^^^

> Rate limiting prevents server overload from too many requests. This is useful to prevent third-party applications or malicous attacks from impacting your server.

Reviewer: ```suggestion
Rate limiting prevents your server from being overloaded with too many requests. This decreases the risk and impact of third-party applications or malicious attacks on your server.
```
